Analysis/include/SyncTupleHTT.h

```diff
@@ -315,6 +324,27 @@ namespace htt_sync {
             sync().jmva_2 = default_value;
         }
 
+        sync().njets_vbf = static_cast<int>(jets_vbf.size());
+        sync().isVBF = jets_vbf.size()>=2 ? true : false;
+        if(jets_vbf.size() >= 1) {
```

Comment:
should fill the selected vbf jets, more correct:
if(vbf_jet_pair.first < jets_vbf.size())

same for the second jet.